rasterfoundry/spec.yml

```diff
@@ -1,100 +1,149 @@
 swagger: '2.0'
 info:
   title: Raster Foundry
-  description: An application to find, view, and analyze geospatial data at any scale
-  version: "0.1.0"
-
+  description: 'An application to find, view, and analyze geospatial data at any scale'
+  version: 0.1.0
 host: app.rasterfoundry.com
-
 schemes:
   - http
   - https
-
 basePath: /api
-
 produces:
   - application/json
 consumes:
   - application/json
-
 x-top-matter:
   - title: Introduction
     level: 1
-    content: |
-      The Raster Foundry API allows developers to find, view, and analyze geospatial data in Raster Foundry. As the name implies, it's especially useful for working with large raster datasets like satellite imagery.
+    content: >
```

Comment:
pretty sure this will break things in the spec - the swagger.io editor defaults to swagger 3.0, and we're using 2.0.

Reply:
Fixed it. I'll use the 2.0 Editor for now on.